scripts/src/indexfile/index.py

```diff
@@ -38,5 +43,97 @@ def get_chart_info(tar_name):
     print(f"[INFO] match not found: {tar_name}")
     return "","","",""
 
+def get_charts_info():
+    chart_info_list = []
+
+    index_dct = _load_index_yaml("https://charts.openshift.io/index.yaml")
+    for entry, charts in index_dct["entries"].items():
+        for chart in charts:
+            chart_info = {}
```

Comment:
It is functionally working fine..  but I think it would have been good to implement .. chart_info as an object of type Chart .. and defining a class `Chart` ... it makes the code more structured

Reply:
For now I like to stick with functionally working fine. Don't have time to rework the code and complete all of the testing especially never having done python object before. Can plan to update in the next story.